src/DDTrace/Tracer.php

```diff
@@ -29,7 +29,7 @@ final class Tracer implements TracerInterface
     /**
      * @deprecated Use Tracer::version() instead
      */
-    const VERSION = '0.29.0'; // Update ./version.php too
+    const VERSION = 'nightly'; // Update ./version.php too
```

Comment:
I like the idea of nightly builds. 
One question: how would we differentiate between different nightly builds across time? 

I.e. a `nightly` version from january will be completely different than `nigthly` from December. 
I was thinking just adding a `nightly` suffix to last released version could work  e.g.  `0.29.0-nigthly`  or maybe some suffix like git sha would as well ?  WDYT?

Reply:
So we had some back and forth on this. And if you  agree this what can be done,  considering that from now on we will no be releasing from master but from a release branch.

1. We update version to `nightly`
2. On a different PR we will create a simple  script to be run in CI/GH action that every time a PR is merged to master it updates the version to be `nightly-20190708-150000` to mark nightly built at 8/7/2019 at 3PM  UTC.

Everybody agree?